RealmSwift/RealmApp.swift

```diff
@@ -0,0 +1,156 @@
+////////////////////////////////////////////////////////////////////////////
```

Comment:
This merge was done incorrectly. The merge path was to add your changes from `RealmApp.swift` to `App.swift` and leave `RealmApp.swift` deleted.